apps/projects/app/components/Content/Settings.js

```diff
@@ -331,7 +299,27 @@ const BountyContractAddress = ({ bountyAllocator }) => (
   </div>
 )
 
-const BaseRate = ({ baseRate, onChange }) => (
+const StyledInputDropDown = styled.div`
```

Comment:
We should avoid duplicating code and use the existing InputDropDown component instead

Reply:
I have tried that, but InputDropDown has predefined Text Input field, and in this case the field is NumericInput instead. Perhaps InputDropDown needs to be generalised to accept arbitrary fields, but it felt out of scope...